packages/surveys/README.md

```diff
@@ -0,0 +1,59 @@
+[![npm version][npm-img]][npm-url]
+[![build status][travis-img]][travis-url]
+[![gzip bundle size][gzip-image]][npm-url]
+[![Coverage Status][coverage-img]][coverage-url]
+[![apache licensed](https://img.shields.io/badge/license-Apache-green.svg?style=flat-square)](https://raw.githubusercontent.com/Esri/hub.js/master/LICENSE)
+
+[npm-img]: https://img.shields.io/npm/v/@esri/hub-surveys.svg?style=flat-square
+[npm-url]: https://www.npmjs.com/package/@esri/hub-surveys
+[travis-img]: https://img.shields.io/travis/Esri/hub.js/master.svg?style=flat-square
+[travis-url]: https://travis-ci.org/Esri/hub.js
+[gzip-image]: https://img.badgesize.io/https://unpkg.com/@esri/hub-surveys/dist/umd/surveys.umd.min.js?compression=gzip
+[coverage-img]: https://codecov.io/gh/Esri/hub.js/branch/master/graph/badge.svg
+[coverage-url]: https://codecov.io/gh/Esri/hub.js
+
+# @esri/hub-surveys
+
+> Module to interact with ArcGIS Hub Surveys in Node.js and modern browsers.
+
+### Example
+
+```bash
+npm install @esri/hub-surveys
+```
+
```

Comment:
Can you provide some common external usage patterns? The API docs will have all the internals, but it may be difficult for most devs to know the few minimal functions to use when working with Hub feedback domain objects.

Reply:
Thanks for pointing out, meant to circle back to this before submitting my PR, will add some helpful docs.